core/storage/classifier/gae_models.py

```diff
@@ -111,3 +113,90 @@ def create(
 
         classifier_model_instance.put()
         return instance_id
+
+
+class ClassifierTrainingJobModel(base_models.BaseModel):
+    """Model for storing classifier training jobs.
+
+    The id of instances of this class has the form
+    {{exp_id}}.{{random_hash_of_16_chars}}
+    """
+
+    # The ID of the algorithm used to create the model.
+    algorithm_id = ndb.StringProperty(required=True, choices=ALGORITHM_CHOICES)
+    # The exploration_id of the exploration to whose state the model belongs.
+    exp_id = ndb.StringProperty(required=True)
+    # The exploration version at the time this training job was created.
+    exp_version_when_created = ndb.IntegerProperty(required=True)
+    # The name of the state to which the model belongs.
+    state_name = ndb.StringProperty(required=True)
+    # The status of the training job. It can be either NEW, COMPLETE or PENDING
+    status = ndb.StringProperty(required=True,
```

Comment:
@anmolshkl I think we need to index this field so that we can easily retrieve the next job to send (check [Fetch Next Job Controller](https://docs.google.com/document/d/1kACmivEvZVlTYBdqxy-mmdTi3sg_NbkZdS10T-ya-pg/edit#heading=h.yu0j0s1lpm2p) for more details on how we retrieve the jobs). So indexing this field will improve retrieval speed.